pkg/cmd/cli/cmd/deploy.go

```diff
@@ -36,7 +37,7 @@ type DeployOptions struct {
 }
 
 const (
-	deployLong = `View, start and restart deployments.
+	deployLong = `View, start, cancel, and retry deployments.
```

Comment:
Should we point out in the `retry` help that the command will be destructive to existing pods and that any pod/container info the user needs should be captured prior to retrying?